### PR TITLE
To ctor

### PR DESCRIPTION
**Pull request by kris-jusiak**
Problem:
-

Solution:
-

Issue: #

Reviewers:
@

### CODE REVIEW

File: extension/include/boost/di/extension/providers/runtime_provider.hpp [hunk 1]
```diff
@@ -58,11 +60,21 @@ class runtime_provider : public config {
   };
 
  public:
+  template <class T>
+  using scope_traits = di::extension::shared_config::scope_traits<T>;
```
Comment (kanstantsin-chernik) [inline extension/include/boost/di/extension/providers/runtime_provider.hpp]: It is better but I still a bit concerned about not being able to configure `runtime_provider`. May be we can have `runitme_provider` as a template to configure scope traits and policies?

File: extension/include/boost/di/extension/providers/runtime_provider.hpp [hunk 2]
```diff
@@ -110,8 +127,15 @@ class injector : public core::injector<runtime_provider> {
   }
 
  private:
-  template <class... Ts, class TInjector>
-  void install(aux::type_list<Ts...>, const TInjector &injector) {
+  template <class TInjector>
+  void install(aux::type_list<>, const TInjector &injector, aux::type<runtime_provider>) {
```
Comment (kanstantsin-chernik) [inline extension/include/boost/di/extension/providers/runtime_provider.hpp]: Just a reminder: it is not thread safe.

Comment (kris-jusiak) [reply extension/include/boost/di/extension/providers/runtime_provider.hpp]: Right :+1:

File: extension/include/boost/di/extension/providers/runtime_provider.hpp (hunk 1 above)
Comment (kris-jusiak) [reply extension/include/boost/di/extension/providers/runtime_provider.hpp]: Done :+1:

**Pull request merged**